So it unfortunately does not build on a Mac:

[ERROR] node-pre-gyp WARN Using request for node-pre-gyp https download 
...
[ERROR] Browserslist: caniuse-lite is outdated. Please run next command `npm update`
...
[INFO] 06 03 2020 17:28:08.195:ERROR [launcher]: Cannot load browser "Safari": it is not registered! Perhaps you are missing some plugin?
[INFO] 06 03 2020 17:28:08.195:ERROR [karma-server]: [Error: Found 1 load error
[INFO]   at Server.webServer.listen (/Users/rrolf/opencast/browser-mac/modules/admin-ui/node_modules/karma/lib/server.js:182:27)
[INFO]   at Object.onceWrapper (events.js:286:20)
[INFO]   at Server.emit (events.js:203:15)
[INFO]   at emitListeningNT (net.js:1313:10)
[INFO]   at process._tickCallback (internal/process/next_tick.js:63:19)
[INFO] ]
[INFO] 06 03 2020 17:28:08.205:INFO [launcher]: Starting browser ChromeHeadless
[INFO] 06 03 2020 17:28:08.548:INFO [launcher]: Starting browser FirefoxHeadless
[INFO] 
[INFO] 
 

Full stack-trace:
https://pastebin.com/rjjPPE18

--- a/modules/admin-ui/package.json
+++ b/modules/admin-ui/package.json
@@ -40,6 +40,7 @@
     "karma-firefox-launcher": "^1.1.0",
     "karma-jasmine": "^1.1.2",
     "karma-ng-html2js-preprocessor": "^1.0.0",
+    "karma-safari-launcher": "^1.0.0",
     "request": "^2.88.0",
     "request-digest": "^1.0.13",
     "selenium-server-standalone-jar": "3.141.5",

--- a/modules/admin-ui/src/test/resources/karma.conf.js
+++ b/modules/admin-ui/src/test/resources/karma.conf.js
@@ -81,16 +81,6 @@ module.exports = function (config) {
             }
         },
 
-        plugins : [
-            'karma-chrome-launcher',
-            'karma-coverage',
-            'karma-detect-browsers',
-            'karma-firefox-launcher',
-            'karma-jasmine',
-            'karma-ng-html2js-preprocessor',
-            'karma-safari-launcher'
-        ],
-
         captureTimeout: 60000,
         browserDisconnectTimeout : 10000,
         browserDisconnectTolerance : 1,

--- a/modules/admin-ui/src/test/resources/karma.conf.js
+++ b/modules/admin-ui/src/test/resources/karma.conf.js
@@ -87,7 +87,8 @@ module.exports = function (config) {
             'karma-detect-browsers',
             'karma-firefox-launcher',
             'karma-jasmine',
-            'karma-ng-html2js-preprocessor'
+            'karma-ng-html2js-preprocessor',
+            'karma-safari-launcher'
         ],
 
         captureTimeout: 60000,

